src/aere_debugger.erl

```diff
@@ -61,11 +61,11 @@ resume_eval(RS, Kind) ->
         {abort, _} ->
             Chain = aere_repl_state:chain_api(RS),
             NewRS = aere_repl_state:set_blockchain_state({ready, Chain}, RS),
-            {aere_msg:contract_exec_ended(), NewRS};
+            {{msg, aere_msg:contract_exec_ended()}, NewRS};
```

Comment:
Can you reflect this change in the type annotation of the function?

Reply:
Done